src/crafting.cpp

```diff
@@ -1336,10 +1338,12 @@ std::vector<item> recipe::create_results(int batch, handedness handed) const
     if( !item::count_by_charges( result ) ) {
         for (int i = 0; i < batch; i++) {
             item newit = create_result(handed);
+            if (contained == true) {newit = newit.in_its_container();}
```

Comment:
The item is already going to be contained here, since that happened in recipe::create_result().
Probably what will happen is in_its_container() doesn't do anything because the plastic bag does not have an assigned container.